content/en/docs/contribute/participating.md

```diff
@@ -32,65 +32,61 @@ aspects of Kubernetes -- the Kubernetes website and documentation.
 
 {{% capture body %}}
 
-## Roles and responsibilities
+## Roles overview
 
-When a pull request is merged to the branch used to publish content (currently
-`master`), that content is published and available to the world. To ensure that
-the quality of our published content is high, we limit merging pull requests to
-SIG Docs approvers. Here's how it works.
+Anyone can contribute to Kubernetes documentation, but only approvers can approve contributions. This keeps contribution quality high. In general:
 
-- When a pull request has both the `lgtm` and `approve` labels and has no `hold`
-  labels, the pull request merges automatically. 
-- Kubernetes organization members and SIG Docs approvers can add comments to
-  prevent automatic merging of a given pull request (by adding a `/hold` comment
-  or withholding a `/lgtm` comment).
-- Any Kubernetes member can add the `lgtm` label, by adding a `/lgtm` comment.
-- Only an approver who is a member of SIG Docs can cause a pull request to merge
-  by adding an `/approve` comment. Some approvers also perform additional
-  specific roles, such as [PR Wrangler](#pr-wrangler) or
-  [SIG Docs chairperson](#sig-docs-chairperson).
-
-For more information about expectations and differences between the roles of
-Kubernetes organization member and SIG Docs approvers, see
-[Types of contributor](/docs/contribute#types-of-contributor). The following
-sections cover more details about these roles and how they work within
-SIG Docs.
+- **Anyone** can contribute to Kubernetes documentation. To contribute, you must [sign the CLA](/docs/contribute/start#sign-the-cla) and have a GitHub account.
+- **Members** of the Kubernetes organization are contributors who have spent time and effort on some area of the Kubernetes project, usually in the form of opening pull requests. See [Community membership](https://github.com/kubernetes/community/blob/master/community-membership.md) for membership criteria.
+- A SIG Docs **Reviewer** is a member of the Kubernetes organization who has
+  expressed interest in reviewing documentation pull requests, and has been
+  added to the appropriate GitHub group and `OWNERS` files in the GitHub
+  repository by a SIG Docs Approver.
+- A SIG Docs **Approver** is a member in good standing who has shown a continued
+  commitment to the project. An approver can merge pull requests
+  and publish content on behalf of the Kubernetes organization.
+  Approvers can also represent SIG Docs in the larger Kubernetes community.
+  Some of the duties of a SIG Docs approver, such as coordinating a release,
+  require a significant time commitment.
 
-### Anyone
+## Anyone
 
-Anyone can file an issue against any part of Kubernetes, including documentation.
+Anyone can do the following:
```

Comment:
Hello @celestehorgan , thanks for the updates. Some suggestions:
nit: lines 37 & 39 are nearly identical
nit: line 39 is identical to `Contribute` index page, Getting Started
nit: line 55, add a new line (list not rendering)
nit: Do you need line 58, (use prow command, `lgtm`) under `Anyone` role? Instead, `Anyone` can provide feedback about the [proposed] changes.

Reply:
Thanks for the feedback! The only one I'll push back on are:

> nit: line 39 is identical to `Contribute` index page, Getting Started
- They do need a little bit of overlap in this case – especially to drive home the important point that anyone can contribute. If we dive straight into the roles and responsibilities portion of this, we'll scare off casual contributions, because they'll think they somehow need to attain membership before contributing :)